bin/i18n/sync-in.rb

```diff
@@ -218,6 +220,17 @@ def localize_level_content
     }
     file.write(I18nScriptUtils.to_crowdin_yaml(formatted_data))
   end
+  File.open(File.join(I18N_SOURCE_DIR, "dashboard/progressions.yml"), 'w') do |file|
```

Comment:
I vote we add this new file as JSON rather than yml! I extracted the "make it possible to consume json" change from my "convert yml to JSON" PR